tools/mpy-tool.py

```diff
@@ -985,6 +1005,75 @@ def freeze_raw_code(self, prelude_ptr=None, type_sig=0):
         raw_code_count += 1
         raw_code_content += 4 * 4
 
+    @staticmethod
+    def decode_lineinfo(line_info: memoryview) -> "tuple[int, int, memoryview]":
```

Comment:
I'm confused why the type annotations are within a string?  Doesn't that render them unusable?

Reply:
Oh, I think I just quoted it on autopilot when I removed the `from __future__ import annotations` line --- putting annotations in quotes is a pure formal equivalent, with the evaluated value of the annotation in CPython just being a string with the annotation body either way.

Reply:
OK, thanks for the explanation.

I understand that type hinting is useful, so I guess it's OK to keep these in here (even though the rest of this file does not have annotations).